integration-libs/punchout/styles/_punchout-template.scss

```diff
@@ -41,6 +41,7 @@
       min-width: 0;
     }
   }
+  min-height: calc(100vh - 1px);
```

Comment:
Doesn't the solution always create a vertical scroll, especially when it's inside an iframe?

Reply:
100vh in ideal conditions is 100% and does not cause a visible vertical scroll yet. I didnt test it in iframe

I'm not saying it's a perfect solution, I see it works fine on inspect page